plugins/github-pull-requests-board/src/components/icons/DraftPr/index.ts

```diff
@@ -14,3 +14,5 @@
  * limitations under the License.
  */
 export { default as DraftPrIcon } from './DraftPr';
+export { default as ArchivedRepoIcon } from './ArchivedRepo';
+
```

Comment:
You may want to turn on auto formatting in your editor, or running `yarn prettier --write <file>` manually on changed files.